spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/server/ManagementServerProperties.java

```diff
@@ -54,6 +54,12 @@ public class ManagementServerProperties {
 	 */
 	private String basePath = "";
 
+	/**
+	 * Enable management access logs prefix customization
+	 * management.server.accesslog.prefix.
+	 */
+	private String accesslogPrefix = "management_";
```

Comment:
Please refactor to use an inner static class named `Accesslog`.

Reply:
done